.jshintrc

```diff
@@ -47,5 +47,5 @@
 	"phantom": true,
 	"rhino": true,
 	"node": true,
-	"esnext": true
+	"esversion": 6
```

Comment:
all of the rest of CanJS is in es5.  So we either:

- need to transpile 
- rewrite in ES5
- make sure all browsers that could support NDJSON support ES6 features being used